target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-mi-router-hd.dts

```diff
@@ -0,0 +1,560 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq8064-v2.0-smb208.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Xiaomi Mi Router HD (R3D)";
+	compatible = "xiaomi,mi-router-hd", "qcom,ipq8064";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x42000000 0x1e000000>;
+	};
+
+	reserved-memory {
+		rsvd@44600000 {
+			reg = <0x44600000 0x200000>; /* IPQ_TZ_APPS_ADDR */
```

Comment:
mhhh why? don't we already reserve this?

Reply:
This was done on the stock firmware. I decided to leave it.

Reply:
please drop or comment this- We don't use trustzone

Reply:
Fixed (drop)